blog/2022-09-15-tauri-1.1.mdx

```diff
@@ -0,0 +1,178 @@
+---
+authors: [lucasfernog]
+image: ./tauri_1_1_images/header.png
+description: After 113 pull requests and nearly two months of work, the Tauri team is pleased to announce the 1.1.0 release.
+---
+
+# Announcing Tauri 1.1.0
+
+![Tauri 1.0 Launch Hero Image](./tauri_1_1_images/header.png)
+
+After 113 pull requests and nearly two months of work, the Tauri team is pleased to announce the 1.1.0 release. The changes were internally audited and no security issues were found.
+
+You can update the NPM dependencies with:
+
+```
+$ npm update
+```
+
+To update the Cargo dependencies, run the following in the `src-tauri` folder:
+
+```
+$ cargo update
+```
+
+## What's in 1.1.0
+
+### Security patch
+
+This release includes a patch for a security vulnerability reported by martin-ocasek. The `readDir` function was able to return entries outside the configured scope when a symlink is found. The patch is also available in 1.0.6. See [the issue](https://github.com/tauri-apps/tauri/issues/4882) for more details.
+
+### Icon generation
+
+We have been recommending using the [tauricon](https://github.com/tauri-apps/tauricon) project to generate icons for your Tauri application using a single source PNG. Several issues have been reported, and we decided to `Rewrite It In Rust` to enhance its stability. This allowed us to move this functionality to the main Tauri CLI, so now you can use the `tauri icon` command.
+
+### cargo-binstall support for the Tauri CLI
+
+The Tauri CLI can now be installed using [cargo-binstall](https://github.com/cargo-bins/cargo-binstall), a mechanism to download and install prebuilt Rust binaries. The binaries are available for the main targets and can be installed with:
+
+```
+$ cargo install cargo-binstall
```

Comment:
maybe remove this and let them follow `cargo-binstall` readme for installation?

Reply:
this is my bad LOL should be just `cargo binstall tauri-cli` but i made a mistake in the CLI cargo.toml metadata for binstall :(

Reply:
then maybe add a note why it should be installed like this until the next release